server/db/common/common.go

```diff
@@ -0,0 +1,63 @@
+// Package common contains utility methods used by all adapters.
+package common
+
+import (
+	"sort"
+	"time"
+
+	t "github.com/tinode/chat/server/store/types"
+)
+
+// SelectEarliestUpdatedSubs selects no more than the given umber of subscriptions from the
```

Comment:
s/he given umber of subscriptions/`numResults`

Reply:
done